### PR TITLE
Update qos_basic_test.go

### PR DESCRIPTION
**Pull request by saarvind6481**
Added Configs for cisco

### CODE REVIEW

File: feature/experimental/qos/ate_tests/qos_basic_test/qos_basic_test.go [hunk 1]
```diff
@@ -66,7 +66,14 @@ func TestBasicConfigWithTraffic(t *testing.T) {
 
 	// Configure DUT interfaces and QoS.
 	ConfigureDUTIntf(t, dut)
-	ConfigureQoS(t, dut)
+	//ConfigureQoS(t, dut)
```
Comment (mohanasm) [inline feature/experimental/qos/ate_tests/qos_basic_test/qos_basic_test.go:69]: You can remove this commented line

File: feature/experimental/qos/ate_tests/qos_basic_test/qos_basic_test.go [hunk 2]
```diff
@@ -409,7 +428,10 @@ func TestBasicConfigWithTraffic(t *testing.T) {
 				counters["dutQosPktsBeforeTraffic"][data.queue] = gnmi.Get(t, dut, gnmi.OC().Qos().Interface(dp3.Name()).Output().Queue(data.queue).TransmitPkts().State())
 				counters["dutQosOctetsBeforeTraffic"][data.queue] = gnmi.Get(t, dut, gnmi.OC().Qos().Interface(dp3.Name()).Output().Queue(data.queue).TransmitOctets().State())
 				counters["dutQosDroppedPktsBeforeTraffic"][data.queue] = gnmi.Get(t, dut, gnmi.OC().Qos().Interface(dp3.Name()).Output().Queue(data.queue).DroppedPkts().State())
-				counters["dutQosDroppedOctetsBeforeTraffic"][data.queue] = gnmi.Get(t, dut, gnmi.OC().Qos().Interface(dp3.Name()).Output().Queue(data.queue).DroppedOctets().State())
+				//counters["dutQosDroppedOctetsBeforeTraffic"][data.queue] = gnmi.Get(t, dut, gnmi.OC().Qos().Interface(dp3.Name()).Output().Queue(data.queue).DroppedOctets().State())
```
Comment (mohanasm) [inline feature/experimental/qos/ate_tests/qos_basic_test/qos_basic_test.go:431]: You can remove this commented line as well

File: feature/experimental/qos/ate_tests/qos_basic_test/qos_basic_test.go [hunk 3]
```diff
@@ -427,7 +449,10 @@ func TestBasicConfigWithTraffic(t *testing.T) {
 				counters["dutQosPktsAfterTraffic"][data.queue] = gnmi.Get(t, dut, gnmi.OC().Qos().Interface(dp3.Name()).Output().Queue(data.queue).TransmitPkts().State())
 				counters["dutQosOctetsAfterTraffic"][data.queue] = gnmi.Get(t, dut, gnmi.OC().Qos().Interface(dp3.Name()).Output().Queue(data.queue).TransmitOctets().State())
 				counters["dutQosDroppedPktsAfterTraffic"][data.queue] = gnmi.Get(t, dut, gnmi.OC().Qos().Interface(dp3.Name()).Output().Queue(data.queue).DroppedPkts().State())
-				counters["dutQosDroppedOctetsAfterTraffic"][data.queue] = gnmi.Get(t, dut, gnmi.OC().Qos().Interface(dp3.Name()).Output().Queue(data.queue).DroppedOctets().State())
+				//counters["dutQosDroppedOctetsAfterTraffic"][data.queue] = gnmi.Get(t, dut, gnmi.OC().Qos().Interface(dp3.Name()).Output().Queue(data.queue).DroppedOctets().State())
```
Comment (mohanasm) [inline feature/experimental/qos/ate_tests/qos_basic_test/qos_basic_test.go:452]: You can remove this commented line as well

File: feature/experimental/qos/ate_tests/qos_basic_test/qos_basic_test.go [hunk 4]
```diff
@@ -66,7 +66,14 @@ func TestBasicConfigWithTraffic(t *testing.T) {
 
 	// Configure DUT interfaces and QoS.
 	ConfigureDUTIntf(t, dut)
-	ConfigureQoS(t, dut)
+	//ConfigureQoS(t, dut)
+	ConfigureDUTIntf(t, dut)
```
[the remark this answers is not in the document]
Comment (saarvind6481) [reply feature/experimental/qos/ate_tests/qos_basic_test/qos_basic_test.go]: Thanks. Removed it

File: feature/experimental/qos/ate_tests/qos_basic_test/qos_basic_test.go (hunk 2 above)
Comment (saarvind6481) [reply feature/experimental/qos/ate_tests/qos_basic_test/qos_basic_test.go]: Thanks . Removed it

File: feature/experimental/qos/ate_tests/qos_basic_test/qos_basic_test.go (hunk 3 above)
Comment (saarvind6481) [reply feature/experimental/qos/ate_tests/qos_basic_test/qos_basic_test.go]: Thanks. Removed it

**Review by prinikasn — COMMENTED** [review]
/gcbrun

**Review by guoshiuan — APPROVED** [review]

**Pull request merged**